src/components/SponsorCallout.astro

```diff
@@ -0,0 +1,44 @@
+---
+import { Image } from 'astro:assets';
+
+interface Props {
+  title: string;
+  logo: ImageMetadata;
+  sponsorName: string;
+  website: string;
+}
+
+const { title, sponsorName, logo, website } = Astro.props;
+
+---
+
+<article class="sl-flex" aria-labelledby={"about-" + sponsorName + "-heading"}>
+  <small id={"about-" + sponsorName + "-heading"}>
+    {title}
+    <span class="sr-only">{ sponsorName }</span>
+  </small>
+  <a href={website} target="_blank"><Image src={logo} alt={ sponsorName + " logo" } rel="noreferrer" /></a>
+  <slot />
+</article>
```

Comment:
It's probably easier to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here. Also, the `rel="noreferrer"` needs to be on the `<a>` tag rather than the image.

```suggestion
<article class="sl-flex" aria-labelledby=`about-${sponsorName}-heading`}>
  <small id=`about-${sponsorName}-heading`}>
    {title}
    <span class="sr-only">{ sponsorName }</span>
  </small>
  <a href={website} target="_blank" rel="noreferrer"><Image src={logo} alt=`${sponsorName} logo`></a>
  <slot />
</article>
```

Reply:
Hah, thanks! That's very useful to know.

> Also, the rel="noreferrer" needs to be on the <a> tag rather than the image.

Oh whoops, that's dumb.